README.md

```diff
@@ -240,12 +305,12 @@ At them moment, only exact matches are supported. However, `false` values do not
 
 ```ts
 mddb.getFiles({
-	frontmatter: {
-		key1: "value1",
-		key2: true,
-		key3: 123,
-		key4: ["a", "b", "c"] // this will match exactly ["a", "b", "c"]
-	}
+  frontmatter: {
```

Comment:
Can we do style guide stuff separately if we can - makes it easier to understand the PR.

Reply:
OK